providers/Bandcamp/mod.ts

```diff
@@ -213,6 +224,13 @@ export class BandcampReleaseLookup extends ReleaseLookup<BandcampProvider, Album
 		}
 		const tracklist = tracks.map(this.convertRawTrack.bind(this));
 
+		if (current.type === 'track') {
+			const track = tracklist[0];
+			if (current.isrc) {
+				track.isrc = current.isrc;
+			}
+		}
```

Comment:
```suggestion
		if (current.type === 'track' && current.isrc) {
			tracklist[0].isrc = current.isrc;
		}
```
Can be simplified now.